pydis_site/templates/events/pages/code-jams/10/frameworks.html

```diff
@@ -0,0 +1,195 @@
+{% extends "events/base_sidebar.html" %}
+
+{% load static %}
+
+{% block title %}Summer Code Jam 2023{% endblock %}
+
+{% block breadcrumb %}
+    <li><a href="{% url "events:index" %}">Events</a></li>
+    <li><a href="{% url "events:page" path="code-jams" %}">Code Jams</a></li>
+    <li><a href="{% url "events:page" path="code-jams/10" %}">Code Jam 2023</a></li>
+    <li class="is-active"><a href="#">Approved Frameworks</a></li>
+{% endblock %}
+
+{% block event_content %}
+    <p>
+        Below is the list of approved frameworks that you can use for the code jam.
+        Please work with your team to choose a library (or multiple) that everyone can and want to develop with.
+        If there is a library not listed below that you think should be here, you're welcome to discuss it with the Events Team over in the Discord server.
+
+    </p>
+    <p>
+        Other libraries can be used to help build your application, but if their primary purpose is image manipulation or processing then they must be approved or from this list.
+        In particular, if you'd like to use a graph/plot creating library similar to MatPlotLib or a machine learning framework/model, double check with us before deciding to use it.
```

Comment:
```suggestion
        In particular, if you'd like to use a graph/plot creating library similar to Matplotlib or a machine learning framework/model, double check with us before deciding to use it.
```
In the docs they seem to capitalise like this in sentences.